lib/services/local/configuration.go

```diff
@@ -243,25 +302,73 @@ func (s *ClusterConfigurationService) GetClusterAuditConfig(ctx context.Context,
 }
 
 // SetClusterAuditConfig sets the cluster audit config on the backend.
-func (s *ClusterConfigurationService) SetClusterAuditConfig(ctx context.Context, auditConfig types.ClusterAuditConfig) error {
-	rev := auditConfig.GetRevision()
-	value, err := services.MarshalClusterAuditConfig(auditConfig)
+func (s *ClusterConfigurationService) SetClusterAuditConfig(ctx context.Context, cfg types.ClusterAuditConfig) error {
+	_, err := s.UpsertClusterAuditConfig(ctx, cfg)
+	return trace.Wrap(err)
+}
+
+// CreateClusterAuditConfig creates a cluster audit config if once does not already exist.
+func (s *ClusterConfigurationService) CreateClusterAuditConfig(ctx context.Context, cfg types.ClusterAuditConfig) (types.ClusterAuditConfig, error) {
+	value, err := services.MarshalClusterAuditConfig(cfg)
 	if err != nil {
-		return trace.Wrap(err)
+		return nil, trace.Wrap(err)
+	}
+
+	item := backend.Item{
+		Key:   backend.Key(clusterConfigPrefix, auditPrefix),
+		Value: value,
+	}
+
+	lease, err := s.Backend.Put(ctx, item)
```

Comment:
```suggestion
	lease, err := s.Backend.Create(ctx, item)
```
To match the other new Create methods.